src/Components/TriggerEditForm/TriggerEditForm.jsx

```diff
@@ -119,11 +126,30 @@ export default class TriggerEditForm extends React.Component<Props, State> {
                     </ValidationWrapperV1>
                 </FormRow>
                 <FormRow label="Description" useTopAlignForLabel>
-                    <Textarea
-                        width="100%"
-                        value={desc || ""}
-                        onChange={(e, value) => onChange({ desc: value })}
-                    />
+                    <div className={cn("tabs-div")}>
+                        <Tabs
+                            value={activeTab}
+                            onChange={(_, val) => this.setState({ activeTab: val })}
+                        >
+                            <Tabs.Tab id="edit">Edit</Tabs.Tab>
+                            <Tabs.Tab id="preview">Preview</Tabs.Tab>
+                        </Tabs>
+                    </div>
+                    {activeTab === "edit" ? (
+                        <Textarea
+                            width="100%"
+                            value={desc || ""}
+                            onChange={(e, value) => {
+                                onChange({ desc: value });
+                            }}
+                        />
+                    ) : (
+                        <div
```

Comment:
Let's write some CSS. 

1. Remove heading styles from `src/style.less`

2. Add to `src/style.less` global classname `.wysiwyg` with styles:
```css
.wysiwyg {
    *:first-child {
        margin-top: 0 !important;
    }

    *:last-child {
        margin-bottom: 0 !important;
    }

    h1 {
        font-weight: 100;
        font-size: 42px;
        line-height: 50px;
    }

    h2 {
        font-weight: 100;
        font-size: 32px;
        line-height: 40px;
    }

    h3 {
        font-weight: normal;
        font-size: 22px;
        line-height: 30px;
    }

    h4,
    h5,
    h6 {
        font-weight: 600;
        font-size: 16px;
        line-height: 25px;
    }

    a {
        color: #3072c4;
        text-decoration: none;

        &:hover {
            text-decoration: underline;
        }
    }

    hr {
        border: none;
        height: 1px;
        background-color: #ccc;
    }

    blockquote {
        margin-left: 0;
        padding-left: 10px;
        border-left: 3px solid #ccc;
        color: #666;
    }
}
```

Reply:
Next add in component less classname `.description-preview`

```css
.description-preview {
    padding: 20px;
    border: 1px dashed #ccc;
}
```

Reply:
And combine it on this `<div className={cn("wysiwyg", "description-preview")} ... />`